ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -9964,6 +10029,8 @@ static bool ggml_backend_vk_device_supports_op(ggml_backend_dev_t dev, const ggm
         case GGML_OP_COUNT_EQUAL:
         case GGML_OP_IM2COL:
         case GGML_OP_TIMESTEP_EMBEDDING:
+        case GGML_OP_CONV_TRANSPOSE_1D:
+            return op->src[0]->type == GGML_TYPE_F32 && op->src[1]->type == GGML_TYPE_F32;
```

Comment:
I think the CI crash happens here - these other ops shouldn't fall through to this logic.

Reply:
Yes, it's definitely an issue.